zookeeper-docs/src/main/resources/markdown/javaExample.md

```diff
@@ -22,10 +22,10 @@ limitations under the License.
     - [Program Design](#sc_design)
   - [The Executor Class](#sc_executor)
   - [The DataMonitor Class](#sc_DataMonitor)
-    - [Note](#note)
+    - [Note](#sc_DataMonitor_Note)
   - [Complete Source Listings](#sc_completeSourceCode)
-    - [Executor.java](#executorjava)
-    - [DataMonitor.java](#datamonitorjava)
+    - [Executor.java](#sc_completeSourceCode_ExecutorJava)
+    - [DataMonitor.java](#sc_completeSourceCode_DataMonitorJava)
 
```

Comment:
Please just fix the typo and don't modify the doc catalogue which is unexpected

Reply:
Ok, I reverted it.